corelib-storage/src/main/java/eu/europeana/corelib/solr/bean/impl/ApiBeanImpl.java

```diff
@@ -106,7 +106,21 @@
     }
 
     @Override
-    public List<Map<String, String>> getEdmPlaceAltLabel() {
+    public List<Map<String,String>> getEdmPlaceAltLabel() {
+        if (edmPlaceAltLabel != null && edmPlaceAltLabel.size() > 0) {
+            List<Map<String, String>> list = new ArrayList<Map<String, String>>();
```

Comment:
Use diamond notation instead
... = new ArrayList<>(edmPlaceAltLabel.size());
Giving the expected size is a performance improvement, try to use this on all collections type constructor